server/monitor_sort_opts_test.go

```diff
@@ -0,0 +1,86 @@
+package server
+
+import (
+	"sort"
+	"testing"
+	"time"
+)
+
+func TestSortByIdleTime(t *testing.T) {
```

Comment:
After the rebase, let's move this back to `monitor_test.go` and rename this test to:
```suggestion
func TestConnzSortByIdleTime(t *testing.T) {
```